imagemounter/disk.py

```diff
@@ -57,6 +57,7 @@ def __init__(self, parser, path, index=None, offset=0, block_size=BLOCK_SIZE, re
         self.rwpath = ""
         self.mountpoint = ""
         self.volumes = VolumeSystem(parent=self, volume_detector=volume_detector, vstype=vstype)
+        self.nbd = False
```

Comment:
I'd rather not have a new public attribute on the ``Disk`` class. Please use ``self._paths`` or auto-detect whether you are unmounting a nbd device in the cleanup phase.